Source/Core/Common/CompatPatches.cpp

```diff
@@ -183,7 +183,7 @@ static bool GetModuleVersion(const wchar_t* name, Version* version)
   if (!data_len)
     return false;
   std::vector<u8> block(data_len);
-  if (!GetFileVersionInfoW(path->c_str(), handle, data_len, block.data()))
+  if (!GetFileVersionInfoW(path->c_str(), 0, data_len, block.data()))
```

Comment:
Is this really necessary? The [docs](https://docs.microsoft.com/en-us/windows/win32/api/winver/nf-winver-getfileversioninfow) say "`dwHandle` This parameter is ignored"; and immediately before the Call to [`GetFileVersionInfoSizeW`](https://docs.microsoft.com/en-us/windows/win32/api/winver/nf-winver-getfileversioninfosizew) indicates that "`lpdwHandle` A pointer to a variable that the function sets to zero."

So, same result, except without using a variable.

Where did you get the "must be zero" from?

Reply:
The tool isn't smart enough to see that it will always be zero (maybe in some cases, it won't be?). In any case, I don't see a problem with just explicitly passing zero.